File: Source/Rio.Web/src/app/shared/models/water-year-quick-open-et-history-dto.ts
import { WaterYearDto } from "./water-year-dto";

export class WaterYearQuickOpenETHistoryDto {
    WaterYear: WaterYearDto;
    CurrentlySyncing: boolean;
    LastSuccessfulSync: Date

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Source/Rio.Models/DataTransferObjects/WaterYearQuickOpenETHistoryDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Rio.Models.DataTransferObjects
{
    public class WaterYearQuickOpenETHistoryDto
    {
        public WaterYearDto WaterYear { get; set; }
        public bool CurrentlySyncing { get; set; }
        public DateTime? LastSuccessfulSync { get; set; }
    }
}
